Parser/asdl_c.py

```diff
@@ -428,6 +428,17 @@ def simpleSum(self, sum, name):
     def buildArgs(self, fields):
         return ", ".join(fields + ["arena"])
 
+    def typeCheck(self, name):
+        self.emit("tp = astmodulestate_global->%s_type;" % name, 1)
+        self.emit("isinstance = PyObject_IsInstance(obj, tp);", 1)
+        self.emit("if (isinstance == -1) {", 1)
+        self.emit("return 1;", 2)
+        self.emit("} else if (isinstance == 0 && field != NULL) {", 1)
+        error = 'field \\"%%s\\" was expecting %s type node, got %%s' % name
```

Comment:
These errors will be a bit weird for someone that only works with the python classes. For instance: `expecting stmt type node` or `was expecting excepthandler type node`. The "type node" also sounds a little odd to me. 

@pganssle could you give us a hand with the English here?

Reply:
Yeah "type node" seems a little weird, though not out of the bounds of native speech.

Maybe it could be improved by "was expecting node of type %s" instead:

"expecting node of type stmt, got excepthandler"

Reply:
Thanks!

Reply:
Thanks Paul, I've applied your suggestion.